steps/run-android-instrumentation-tests/1.0.0/step.yml

```diff
@@ -0,0 +1,136 @@
+    is_expand: true
+    is_required: true
+    summary: The path to the APK to test.
+    title: APK path
+- opts:
+    description: |
+      The path to the test APK that contains instrumentation tests.
+
+      You can easily generate the test APK by running the [android-build-for-ui-testing](https://bitrise.io/integrations/steps/android-build-for-ui-testing) Step before this Step. In this case, the test APK path is exported under the `$BITRISE_TEST_APK_PATH` environment variable.
+    is_expand: true
+    is_required: true
+    summary: The path to the test APK that contains instrumentation tests.
+    title: Test APK path
+  test: $BITRISE_TEST_APK_PATH
+- num_shards: 1
+  opts:
+    description: |
+      Tests are split evenly by count across shards and run in parallel.
+
+      In general, the more shards you use, the faster your total test run time will be. Make sure not to set this count to be bigger than your actual test count.
+    summary: Tests are split evenly by count across shards and run in parallel.
+    title: Number of test shards
+- device: null
+  opts:
+    description: |
+      Device configuration where to run tests.
+
+      This is only required when you don't want to run tests on the default emulator configuration.
+
+      The value of this parameter must be a list of key=value pairs where supported keys are:
+
+      -  `model`: device model to use, possible values [`Pixel2`, `NexusLowRes`]
+      -  `version`: API version to use
+
+      For example, to test on an low res MDPI device with API 23 specify `model=NexusLowRes,version=23`
+    summary: Device configuration where to run tests
+    title: Device configuration
+- opts:
+    title: Use Android Test Orchestrator
+    value_options:
+    - "yes"
+    - "no"
+  use_orchestrator: "yes"
+- clear_package_data: "yes"
+  opts:
+    summary: Only applicable when using Android Test Orchestrator
+    title: Clear package data between runs
+    value_options:
+    - "yes"
+    - "no"
+- opts:
+    title: Collect code coverage data while executing the test
+    value_options:
+    - "yes"
+    - "no"
+  with_coverage: "no"
+- opts:
+    summary: Using $BITRISE_TEST_RESULT_DIR here will make sure that they are also
+      available under the Test Reports view
+    title: Output path where test run outputs (junit XML, logcat, files) will be stored.
+  outputs_dir: $BITRISE_TEST_RESULT_DIR
+- additional_apks: null
+  opts:
+    summary: A comma separated list of additional APKs to install, in addition to
+      those being directly tested.
+    title: Additional APKs to install
+- environment_variables: null
+  opts:
+    description: |
+      Comma separated list of key=value pairs that are passed as extra options to the `am instrument` command.
```

Comment:
Pipe should be used as a separator, not the comma: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#inputs-which-can-accept-a-list-of-values